python/ray/autoscaler/aws/example-cloudwatch.yaml

```diff
@@ -0,0 +1,116 @@
+
+            # This tag serve as an unique identifier to ami name with Cloudwatch agent installed.
+            # Add this tag to your customized AMI name with cloudwatch agent installed.
+            CLOUDWATCH_AGENT_INSTALLED_AMI_TAG: "T6Iq2faj"
+            config: "cloudwatch/example-cloudwatch-agent-config.json"
+            retryer:
+                # Max allowed CloudWatch agent installation attempts on any host.
+                max_attempts: 120
+                # Seconds to wait between each CloudWatch agent installation attempt.
+                delay_seconds: 30
+        dashboard:
+            # CloudWatch Dashboard name
+            name: "example-dashboard-name"
+            # Path to the CloudWatch Dashboard config file
+            # The CloudWatch Dashboard config file contains dashboard widgets to display for your cluster.
+            # See https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html for config file syntax.
+            config: "cloudwatch/example-cloudwatch-dashboard-config.json"
+        alarm:
+            # Path to CloudWatch Alarm config file
+            # The CloudWatch alarm config file is a JSON file specifying all alarms you want to add.
+            # See https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html for additional details.
+            # Note: To allow per-node alarm being created and updated, we recommend "{instance_id}" included as part of "AlarmName" in the following json config.
+            config: "cloudwatch/example-cloudwatch-alarm-config.json"
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ubuntu
+
+available_node_types:
+  ray.head.default:
+      node_config:
+        InstanceType: c5a.large
+        ImageId: latest_dlami
+      resources: {}
+      max_workers: 10
+  ray.worker.default:
+      node_config:
+        InstanceType: c5a.large
+        ImageId: latest_dlami
+        # Note: IamInstanceProfile is needed to grant worker nodes required permission to make boto3 call for Cloudwatch setup.
+        # Default IamInstanceProfile is arn:aws:iam::{your_aws_account_number}:instance-profile/ray-autoscaler-cloudwatch-v1.
+        IamInstanceProfile:
+          Arn: arn:aws:iam::{your_aws_account_ID}:instance-profile/ray-autoscaler-cloudwatch-v1
+      resources: {}
+      min_workers: 1
+head_node_type: ray.head.default
+
+# Files or directories to copy to the head and worker nodes. The format is a
+# dictionary from REMOTE_PATH: LOCAL_PATH, e.g.
+# If you want to export Ray system-level Prometheus metrics to Cloudwatch,
+# Uncomment below commands under file_mounts and head_setup_commands.
+# Commands under these two sections all serve for exporting Prometheus metrics to
+# CloudWatch. See https://docs.ray.io/en/master/ray-metrics.html for Ray prometheus support.
+#file_mounts: {
+#    "~/cloudwatch/prometheus.yml": "cloudwatch/prometheus.yml",
```

Comment:
I think the path on the right needs to be absolute, otherwise it will be taken as relative to the directory from which ray up is called.
If prometheus.yml is meant to be configured, then including it here with an absolute path will work.
If it's not meant to be configured, then including it in setup.py may be better.